hack/tests/scaffolding/e2e-go-scaffold.sh

```diff
@@ -4,6 +4,24 @@ set -ex
 
 source hack/lib/test_lib.sh
 
+function download_old_sdk_binary() {
+    VERSION="v0.18.1"
+    echo "Getting SDK $VERSION release binary..."
+    BIN_URL=""
+    OS=$(go env GOOS)
+    if [ "$OS" = "darwin" ]; then
+        BIN_URL="https://github.com/operator-framework/operator-sdk/releases/download/$VERSION/operator-sdk-$VERSION-x86_64-apple-darwin"
+    elif [ "$OS" = "linux" ]; then
+        BIN_URL="https://github.com/operator-framework/operator-sdk/releases/download/$VERSION/operator-sdk-$VERSION-x86_64-linux-gnu"
+    else
+        echo "Failed to get SDK $VERSION release binary for $OS"
+        exit 1
+    fi
+
+    curl -o operator-sdk-old -L $BIN_URL
```

Comment:
Nit (resolves @joelanford's [comment](https://github.com/operator-framework/operator-sdk/pull/3190/files#r440561147) IMO):
```suggestion
    mkdir -p build
    curl -o ./build/operator-sdk-old -L $BIN_URL
```

Reply:
Addressed that comment above, so this should be fine.